app/scripts/setup.sh

```diff
@@ -74,18 +74,24 @@ function createSymlinks() {
     esac
 }
 
-function checkoutSilme() {
-    # Check out SILME library to a specific version (0.8.0)
-    if [ ! -d $libraries/silme/.hg ]
+function setupExternalLibraries() {
+    # Check out or update compare-locales library
+    version="RELEASE_1_2_1"
```

Comment:
Yeah you’re right, we don’t want to debug instances where people would see “weird bugs” because they are using the wrong compare-locales version. I was only thinking about the prod server.
Let’s keep what you did, we can still revisit later if it happens we need to update it more frequently